### ISSUE TITLE
Learning rate

### ISSUE THREAD

**Issue by akumar14** [bearpaw/pytorch-pose#45]
Why is the initial learning rate kept so low in the implementation ? A learning rate of 1e-3 is not yieldin NaNs. Is it an observation that low learning rates work well for pose estimation tasks ? 

What is the accuracy that is being printed ? Is it just the distance between target and predicted masks? In that case lower accuracy should mean closer to gt.

**Comment by bearpaw** [comment]
You are welcome to try other LR and compare the performance and share your results with us!

The accuracy here is the PCK evaluation metric for pose estimation. Details can be found in http://human-pose.mpi-inf.mpg.de/#results
